rdkit/Chem/UnitTestPandasTools.py

```diff
@@ -166,7 +288,7 @@ def test_specify_numeric_column(self):
     self.assertIn("7\n\n", s)
     self.assertIn("8\n\n", s)
 
-  def test_specify_numeric_column(self):
+  def test_specify_numeric_column_2(self):
```

Comment:
The method name `test_specify_numeric_column` was used twice. The tests look fairly similar, so one might be enough.